reverse_engineering/helpers/snowflakeHelper.js

```diff
@@ -1329,15 +1331,54 @@ const getContainerData = async schema => {
 			sequences,
 			fileFormats,
 			isCaseSensitive,
+			tags,
+			schemaTags,
 		};
 		containers[schema] = data;
 
 		return data;
-	} catch (err) {
+	} catch (error) {
+		logger.log('error', { error }, 'Reverse Engineering error while retrieving schema data');
+
 		return {};
 	}
 };
 
+const getTagAllowedValues = ({ values, logger }) => {
+	try {
+		if (typeof values !== 'string') {
+			return [];
+		}
+		const allowedValues = JSON.parse(values);
+		return allowedValues.map(value => ({ value }));
+	} catch (error) {
+		logger.log('error', { error }, 'Reverse Engineering error while retrieving tag allowed values');
+
+		return [];
+	}
+};
+
+const getTags = async ({ dbName, schemaName, logger }) => {
+	const rows = await execute(`SHOW TAGS IN SCHEMA "${removeQuotes(dbName)}"."${removeQuotes(schemaName)}";`);
```

Comment:
I would add error hangling here, the same as you did for `getTagAllowedValues`.
So RE can continue in case the error happans while getting tags.